internal/serve/httphandler/disbursement_handler.go

```diff
@@ -162,8 +163,7 @@ func (d DisbursementHandler) GetDisbursements(w http.ResponseWriter, r *http.Req
 	}
 
 	ctx := r.Context()
-	disbursementManagementService := services.NewDisbursementManagementService(d.Models, d.DBConnectionPool)
-	resultWithTotal, err := disbursementManagementService.GetDisbursementsWithCount(ctx, queryParams)
```

Comment:
💡  You can also remove the DBConnectionPool parameter from `DisbursementHandler`, since you can access the connection pool from the Models object.